media/inf/flpydisk.inf

```diff
@@ -82,7 +82,7 @@ Floppy.DeviceDesc = "Disquetera genérica"
 ReactOS = "Zespół ReactOS"
 
 FloppyClassName = "Stacje dyskietek"
-GenericMfg = "(Standardowe stacje dyskietek)"
+GenericMfg = "(Standardowe sterowniki stacji dyskietek)"
```

Comment:
Original translation is correct. See english translation for reference:

> (Generic floppy disk drives)

**Drives**, not drivers.

Reply:
its description from WinNT 5.1+ :)

Reply:
We are not copying translations from Windows as is, because it's copyrighted material.